include/crm_internal.h

```diff
@@ -38,6 +38,7 @@
 
 #include <crm/lrmd.h>
 #include <crm/cluster/internal.h>
+#include <crm/common/cmdline_internal.h>
```

Comment:
This will come as no surprise to you, but our include files are a mess.  A lot of the internal includes in this list are also included by `crm/common/internal.h` so they could simply be removed.  As far as I remember, there's no particular logic behind what is and is not included in that file.

It may be worth one of us one day doing a PR that just straightens all this stuff out.  Ugh.